table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -253,15 +314,19 @@ public UdbTable getTable(String tableName, UdbBypassSpec bypassSpec) throws IOEx
         }
       }
 
-      PathTranslator pathTranslator = mountAlluxioPaths(table, partitions, bypassSpec);
+      if (!mIsMounted) {
```

Comment:
This condition check is not thread-safe, suggest to move this mount logic before invoke tasks.